### PR TITLE
New package Aircall.Aircall v3.1.8

### PR DESCRIPTION
**Pull request by FaserF**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/138531)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-02-09T14:10:53.9127213Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[PackageUrl] Description: Failed to match regex specified by 'pattern' constraint. Error context: <root> Description: Failed to validate against schema associated with property name 'PackageUrl'. ] File: Aircall.Aircall.locale.en-US.yaml

(Automated response - build 720.)

File: manifests/a/Aircall/Aircall/3.1.8/Aircall.Aircall.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+PackageIdentifier: Aircall.Aircall
+PackageVersion: 3.1.8
+PackageLocale: en-US
+Publisher: Aircall
+PublisherUrl: https://aircall.io/business-phone-system/
+PublisherSupportUrl: https://aircall.io/inbound-call-center-software/
+PrivacyUrl: https://aircall.io/privacy/
+Author: Aircall
+PackageName: Aircall
+PackageUrl: A flexible solution that fits the teams you have today (and the plans you have for tomorrow).
```
Comment (stephengillie) [inline manifests/a/Aircall/Aircall/3.1.8/Aircall.Aircall.locale.en-US.yaml:13]:
```suggestion
PackageUrl: https://aircall.io/download/
```

Comment (FaserF) [reply manifests/a/Aircall/Aircall/3.1.8/Aircall.Aircall.locale.en-US.yaml:13]: Whoops, thanks.

**Comment by FaserF** [comment]
> Url validation error
> 
> * manifests\a\Aircall\Aircall\3.1.8
>   
>   * https://support.aircall.io/hc/en-gb/sections/10393020321821-Release-Notes
>     
>     * Http status code: Forbidden

Should I remove the Releasenotes URL then? For me it looks fine, can open the link without issues on any device without a login and without a forbidden.

**Comment by stephengillie** [comment]
> Should I remove the Releasenotes URL then? For me it looks fine, can open the link without issues on any device without a login and without a forbidden.

Yes, please do. This might be due to a network block of data centers, to prevent automated downloads.

File: manifests/a/Aircall/Aircall/3.1.8/Aircall.Aircall.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+PublisherUrl: https://aircall.io/business-phone-system/
+PublisherSupportUrl: https://aircall.io/inbound-call-center-software/
+PrivacyUrl: https://aircall.io/privacy/
+Author: Aircall
+PackageName: Aircall
+PackageUrl: https://aircall.io/download/
+License: Proprietary
+Copyright: Copyright © 2023, Aircall
+ShortDescription: A flexible solution that fits the teams you have today (and the plans you have for tomorrow).
+ReleaseNotesUrl: https://support.aircall.io/hc/en-gb/sections/10393020321821-Release-Notes
```
Comment (stephengillie) [inline manifests/a/Aircall/Aircall/3.1.8/Aircall.Aircall.locale.en-US.yaml:17]:
```suggestion
# ReleaseNotesUrl: 
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 726.)

File: manifests/a/Aircall/Aircall/3.1.8/Aircall.Aircall.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+InstallerType: msi
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /install /quiet /norestart
+  SilentWithProgress: /install /quiet /norestart
+  Custom: /install /quiet /norestart
```
Comment (stephengillie) [inline manifests/a/Aircall/Aircall/3.1.8/Aircall.Aircall.installer.yaml:16]:
```suggestion
  Silent: /quiet /norestart
  SilentWithProgress: /passive /norestart
```

**Comment by FaserF** [comment]
Hmm, I dont get the issue to be honest. Since it's a MSI it should be pretty simple. 
installing it via Terminal on my machine with .\Aircall-3.1.8.msi /quiet /norestart works fine. And I dont get from the Pipeline logs where the issue is.

**Comment by FaserF** [comment]
Hmm, I dont get the issue to be honest. Since it's a MSI it should be pretty simple.
installing it via Terminal on my machine with .\Aircall-3.1.8.msi /quiet /norestart works fine. And I dont get from the Pipeline logs where the issue is.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978334957 2024-02-12 05:20:00.046 [CLI ] ShellExecute installer failed: 1633 MSI (s) (D8:EC) [05:19:59:643]: Product: Aircall -- Installation failed. MSI (s) (D8:EC) [05:19:59:643]: Windows Installer installed the product. Product Name: Aircall. Product Version: 3.1.8.3897. Product Language: 1033. Manufacturer: Aircall. Installation success or error status: 1633.

(Automated response - build 729.)

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
> Hmm, I dont get the issue to be honest. Since it's a MSI it should be pretty simple. installing it via Terminal on my machine with .\Aircall-3.1.8.msi /quiet /norestart works fine. And I dont get from the Pipeline logs where the issue is.

The pipeline logs are a little buried: 

First steps (common for both):
- Goto the `Checks` tab, expand `Azure Pipelines`, click `WinGetSvc-Validation`. Alternatively, click one of the `Details` links in the Checks section that sometimes shows between these comments and the comment box below.
- Click `View more details on Azure Pipelines`

For Run Artifact Logs:
- Under `Related` find `4 published; 1 consumed` or similar. (Might say `3 published` etc.)
- Find `installationVerificationLogs` and follow the row to the right edge, then click the 3 vertical dots, and click `Download artifacts`. 

For ADO logs: 
- Click `WinGetSvc_Validation` in the rectangle under "Stages".
- Click through red X sections to find the error. 
- Note: Not all errors from this system are emitted in this way. Sometimes these results will be inconclusive.

To streamline this, I've built out some [scripted automation](https://github.com/microsoft/winget-pkgs/blob/3125655cdbcad121815918494d1cc60b3707353a/Tools/ManualValidationPipeline.ps1#L992) to retrieve it into the PR comment thread.

**Pull request merged**